build_e2b_image.sh

```
@@ -0,0 +1,22 @@
+# Check if argument is provided
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)